ingress/controllers/gce/controller/utils.go

```diff
@@ -37,6 +38,10 @@ import (
 const (
 	allowHTTPKey    = "kubernetes.io/ingress.allow-http"
 	staticIPNameKey = "kubernetes.io/ingress.global-static-ip-name"
+
+	// Label key to denote which GCE zone a Kubernetes node is in.
+	zoneKey     = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
this is the zone label, not the ubernetes federation zone afaik. It should show up on every non-ubernetes node too with kubectl get no --show-labels.